managedTokens.yml

```diff
@@ -72,7 +72,7 @@ loki:
 
 # Tracing settings
 tracing:
-  url: hostname.domain # Required for tracing use
+  endpoint: scheme://hostname.domain # Required for tracing use
```

Comment:
1) In general I'm not a big fan of breaking config changes, but as the sole deployment it's not a stopper.

2) The endpoint is in fact a URL now, so either really is ok.

Reply:
Oh that's a good point.  We had the discussion and I forgot to change this back.  I'll do that now across the board.

Reply:
Done.